Test plan Final version and test cases

NanaChocolat opened the pull request:
Add the final version of the test plan.

thomas-planchard committed:
Update Technical specification

thomas-planchard committed:
Update TechnicalSpecifications.md

Instruction format, opcode

HiNett committed:
Update FunctionalSpecifications.md

Update the Functional Specifications in the Documents branch with the actual one in the main.

thomas-planchard committed:
Update technical

Refactor clean and tokenization part

HiNett committed:
Update the Documents Branch

Update the Branch to match with the Main

Clementine951 committed:
Move the Jira tutorial

Clementine951 committed:
Update of the tutorial

- add images
- modify the links

Clementine951 committed:
Correction of grammar errors

- add small text to the tutorial
- correction of grammar errors

thomas-planchard committed:
Update TechnicalSpecifications.md

thomas-planchard committed:
Update TechnicalSpecifications.md

thomas-planchard committed:
Update TechnicalSpecifications.md

thomas-planchard committed:
Update TechnicalSpecifications.md

AlexisLasselin committed:
Creating the  dvelopment process

thomas-planchard committed:
Update Technical

Parsing + development process

thomas-planchard committed:
Update TechnicalSpecifications.md

NanaChocolat committed:
Update test plan

thomas-planchard committed:
Update technical, add semantic.png

Error handling system done
Semantic analysis done
glossary refactor

thomas-planchard committed:
Update TechnicalSpecifications.md

thomas-planchard committed:
Update TechnicalSpecifications.md

Same errors code with the functional
Machine code generation

thomas-planchard committed:
Update TechnicalSpecifications.md

thomas-planchard committed:
Update TechnicalSpecifications.md

First version DONE

NanaChocolat committed:
update test plan and add test cases file

NanaChocolat committed:
updates

NanaChocolat committed:
Update TestCases.md

Clementine951 commented:
Sorry, I can't accept the pull request, there are conflicts with other documents
Documents/Appendices/semantic.png
Documents/FunctionalSpecifications.md
Documents/TechnicalSpecifications.md
README.md

HiNett committed:
Update Branch for the Pull Request

Added the branch to the pull request so that it can be merged without conflicts.

Clementine951 reviewed (approved):
I didn't have the time to review the document, but knowing the deadline is in 2 minutes, I approve.

HiNett reviewed (approved):
Due to lack of time for review, I approve without any review.
